### ISSUE TITLE
What and where is the Loss function used in LLAVA-MED

### ISSUE THREAD

**Issue by peterphancong** [microsoft/LLaVA-Med#54]
Hi authors,
I am trying to finetune the LLAVA-MED with my custom classification purpose, and I would like to know what is the loss function used in LLAVA-MED, and is it possible to update for my custom training?
